_includes/live-demos/premiumskinsandicons-fabric/style.css

```diff
@@ -5,7 +5,7 @@
 }
 
 .dummy-header {
-    background-color: #2b579a;
+    background-color: {{site.primaryColor}};
```

Comment:
This seems incorrect, as this is meant to simulate a windows looking editor, so we wouldn't want that using out blue.

Reply:
Spoke during a meeting. We would like to change this example from "MS Word" blue to "Tiny Blue"